global/scripts/MSMSulc.sh

```diff
@@ -0,0 +1,117 @@
+		Structure="CORTEX_LEFT"
+	elif [[ "$Hemisphere" == "R" ]] ; then
+		Structure="CORTEX_RIGHT"
+	fi
+	  
+	#convert relative to absolute for correctness after cd, restrict HEMISPHERE replacement to filename only
+	RefMeshFile=$(basename -- "$RefMesh")
+	ReferenceMesh="$(cd "$(dirname -- "$RefMesh")"; pwd)/${RefMeshFile/HEMISPHERE/$Hemisphere}"
+	RefDataFile=$(basename -- "$RefData")
+	ReferenceData="$(cd "$(dirname -- "$RefData")"; pwd)/${RefDataFile/HEMISPHERE/$Hemisphere}"
+	(
+		cd "$NativeFolder"/"$RegName"
+
+		#Register using FreeSurfer Sulc Folding Map Using MSM Algorithm Configured for Reduced Distortion
+		"$MSMBINDIR"/msm --conf="$ConfFile" --inmesh="$NativeFolder"/"$Subject"."$Hemisphere".sphere.rot.native.surf.gii --refmesh="$ReferenceMesh" --indata="$NativeFolder"/"$Subject"."$Hemisphere".sulc.native.shape.gii --refdata="$ReferenceData" --out="$NativeFolder"/"$RegName"/"$Hemisphere". --verbose
+	)
+
+	cp "$ConfFile" "$NativeFolder"/"$RegName"/"$Hemisphere".logdir/conf
+	cp "$NativeFolder"/"$RegName"/"$Hemisphere".sphere.reg.surf.gii "$NativeFolder"/"$Subject"."$Hemisphere".sphere."$RegName".native.surf.gii
+	
+	wb_command -set-structure "$NativeFolder"/"$Subject"."$Hemisphere".sphere."$RegName".native.surf.gii "$Structure"
+
+	wb_command -surface-distortion "$NativeFolder"/"$Subject"."$Hemisphere".sphere.native.surf.gii "$NativeFolder"/"$Subject"."$Hemisphere".sphere."$RegName".native.surf.gii "$NativeFolder"/"$Subject"."$Hemisphere".ArealDistortion_"$RegName".native.shape.gii
+	wb_command -set-map-names "$NativeFolder"/"$Subject"."$Hemisphere".ArealDistortion_"$RegName".native.shape.gii -map 1 "$Subject"_"$Hemisphere"_Areal_Distortion_"$RegName"
+	wb_command -metric-palette "$NativeFolder"/"$Subject"."$Hemisphere".ArealDistortion_"$RegName".native.shape.gii MODE_AUTO_SCALE -palette-name ROY-BIG-BL -thresholding THRESHOLD_TYPE_NORMAL THRESHOLD_TEST_SHOW_OUTSIDE -1 1
+
+	wb_command -surface-distortion "$NativeFolder"/"$Subject"."$Hemisphere".sphere.native.surf.gii "$NativeFolder"/"$Subject"."$Hemisphere".sphere."$RegName".native.surf.gii "$NativeFolder"/"$Subject"."$Hemisphere".EdgeDistortion_"$RegName".native.shape.gii -edge-method
+	wb_command -surface-distortion "$NativeFolder"/"$Subject"."$Hemisphere".sphere.native.surf.gii "$NativeFolder"/"$Subject"."$Hemisphere".sphere."$RegName".native.surf.gii "$NativeFolder"/"$Subject"."$Hemisphere".Strain_"$RegName".native.shape.gii -local-affine-method -log2
+
+	wb_command -metric-merge "$NativeFolder"/"$Subject"."$Hemisphere".StrainJ_"$RegName".native.shape.gii -metric "$NativeFolder"/"$Subject"."$Hemisphere".Strain_"$RegName".native.shape.gii -column 1
+	wb_command -metric-merge "$NativeFolder"/"$Subject"."$Hemisphere".StrainR_"$RegName".native.shape.gii -metric "$NativeFolder"/"$Subject"."$Hemisphere".Strain_"$RegName".native.shape.gii -column 2
+
+	rm "$NativeFolder"/"$Subject"."$Hemisphere".Strain_"$RegName".native.shape.gii
+done
+
+if [[ "$Hemi" == "L R" ]] ; then
```

Comment:
```suggestion
if [[ "$Hemi" == *L* && "$Hemi" == *R* ]] ; then
```
Keep behavior consistent even if someone reverses them, or adds extra whitespace, or...